core/net/net.go

```diff
@@ -30,38 +30,41 @@ type API interface {
 	// GetHostID returns the host's peer id.
 	GetHostID(ctx context.Context) (peer.ID, error)
 
+	// GetToken returns a signed token representing an identity.
+	GetToken(ctx context.Context, identity thread.Identity) (thread.Token, error)
```

Comment:
This is the "login" method that returns a stateless token (JWT) representing an "identity".